cli/ingress-controller/main.go

```diff
@@ -385,7 +385,7 @@ func main() {
 
 	kongIngressInformer := kongInformerFactory.Configuration().V1().KongIngresses().Informer()
 	kongIngressInformer.AddEventHandler(reh)
-	cacheStores.Configuration = kongIngressInformer.GetStore()
+	cacheStores.KongIngress = kongIngressInformer.GetStore()
```

Comment:
note: while working on this the `Configuration` naming as the cache store for `KongIngress` seemed confusing, it seemed to make more sense to just refer to it as the `KongIngress` cache so I made this change here and below.